lib/tapioca/cli.rb

```diff
@@ -114,60 +123,75 @@ def dsl(*constants)
     end
 
     desc "gem [gem...]", "generate RBIs from gems"
+    option :outdir,
+      aliases: ["--out", "-o"],
+      banner: "directory",
+      desc: "The output directory for generated RBI files",
+      default: DEFAULT_GEMDIR
+    option :file_header,
+      type: :boolean,
+      desc: "Add a \"This file is generated\" header on top of each generated RBI file",
+      default: true
     option :all,
       type: :boolean,
-      default: false,
-      desc: "Regenerate RBI files for all gems"
+      desc: "Regenerate RBI files for all gems",
+      default: false
     option :prerequire,
       aliases: ["--pre", "-b"],
       banner: "file",
-      desc: "A file to be required before Bundler.require is called"
+      desc: "A file to be required before Bundler.require is called",
+      default: nil
     option :postrequire,
       aliases: ["--post", "-a"],
       banner: "file",
-      desc: "A file to be required after Bundler.require is called"
+      desc: "A file to be required after Bundler.require is called",
+      default: DEFAULT_POSTREQUIRE
     option :exclude,
       aliases: ["-x"],
       type: :array,
       banner: "gem [gem ...]",
-      desc: "Excludes the given gem(s) from RBI generation"
+      desc: "Excludes the given gem(s) from RBI generation",
```

Comment:
Let's be consistent with the options for `dsl`
```suggestion
      desc: "Exclude the given gem(s) from RBI generation",
```